README.md

```diff
@@ -760,6 +760,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Keen IO](https://keen.io/) | Data Analytics | `apiKey` | Yes | Unknown |
 | [NLP Cloud](https://nlpcloud.io) | NLP API using spaCy and transformers for NER, sentiments, classification, summarization, and more | `apiKey` | Yes | Unknown |
 | [OpenVisionAPI](https://openvisionapi.com) | Open source computer vision API based on open source models | No | Yes | Yes |
+| [Perspective](https://perspectiveapi.com) | NLP API to return probability that if text is toxic, obscene, insulting or threatening | Yes | Yes | Unknown |
```

Comment:
Fix this:

auth value is not enclosed with \`backticks\`
Yes is not a valid Auth option

Currently, the only accepted inputs for the Auth field are as follows:

`OAuth` - the API supports OAuth
`apiKey` - the API uses a private key string/token for authentication - try and use the correct parameter
`X-Mashape-Key` - the name of the header which may need to be sent
`User-Agent` - the name of the header to be sent with requests to the API
No - the API requires no authentication to run

```suggestion
| [Perspective](https://perspectiveapi.com) | NLP API to return probability that if text is toxic, obscene, insulting or threatening | `apiKey` | Yes | Unknown |
```
ref: https://developers.perspectiveapi.com/s/docs-enable-the-api